src/components/masters/program/ProgramPage.jsx

```diff
@@ -118,10 +118,10 @@ class ProgramPage extends Component {
                   </main>
                 </>
               ) : (
-                this.renderError()
-              )}
+                  this.renderError()
+                )}
             </>
-          )}
+            )}
```

Comment:
super nit: It looks like this is pulling this out of line? It might just be the linter being cranky though, and it's not particularly important.

Reply:
ahhh its prettier -- I must have accidentally turned it back on for one of these commits. It collides with the linter config and causes weird issues like this.